src/content/mentees/renan-garcia.md

```diff
@@ -0,0 +1,29 @@
+---
+name: 'Renan Garcia'
```

Comment:
Hi @rengarcia! Good one for submitting your task early ✋🏽

Could you please submit a separate pull request with just your info added to the mentees list? Your other contributions can go in a separate PR. I have added the structure to the repo now.